GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -440,6 +442,15 @@ protected override void Dispose(bool disposing)
         {
             if (disposing)
             {
+                if (_splitterManager is null)
+                {
+                    // UICommands is unset in TranslationApp
+                }
+                else
+                {
```

Comment:
```suggestion
                if (UICommands is not null)
                {
```
Checking the seemingly unrelated `_splitterManager` seems hacky. If this is just to prevent a null exception, then the check should be direct.

Reply:
Not possible because the property `UICommands` throws if `null`. But I have decoupled the ViewModel from UICommands.